frontend/containers/image-gallery/component.tsx

```diff
@@ -0,0 +1,77 @@
+import { FC, useState } from 'react';
+
+import { useIntl } from 'react-intl';
+
+import cx from 'classnames';
+
+import Image from 'next/image';
+
+import Button from 'components/button';
+import Carousel, { Slide } from 'components/carousel';
+import Modal from 'components/modal';
+
+import { ImageGalleryProps } from '.';
+
+export const ImageGallery: FC<ImageGalleryProps> = ({ images }) => {
+  // Sets the index of the image displayed on carousel. The default is undefined for not displaing any image.
+  const [active, setActive] = useState<number>();
+  const { formatMessage } = useIntl();
+
+  return (
+    <div area-hidden>
+      <ul className="flex gap-1">
```

Comment:
I'd add a `role="group"` and an `aria-label` with something like: _"Project images"_ (w/ `react-intl`) for accessibility purposes.

Testing with VoiceOver, the screen reader would read "Project images, group", making it clearer for the user when they move onto the image buttons that they are a group of project images buttons.

Reply:
After talking with Clement we decided to hide (area-hidden)  the component, because it will not add anything  to a screen reader.